source/_assets/less/markdown.less

```diff
@@ -1,4 +1,4 @@
-.documentation-page {
+.markdown {
```

Comment:
This seems unnecessary...I'd just add these two to the single `div` that uses the `.markdown` class and remove this.